### ISSUE TITLE
Continue to finalize designs for "events near me" research with editors

### ISSUE THREAD

**Issue by fmccaf1** [department-of-veterans-affairs/va.gov-cms#10484]
## Description
Take next steps for designing "events near me" prototype for editor research.

## Acceptance Criteria
- [ ] Meet with Cindy to discuss possible new components incl: 3-button toggle and view-only/read-only location box
- [ ] Meet with Wes to discuss learnings thus far, poke holes in design, think through work-flows
- [ ] Request and receive content audits from CMS team: 1) Non-VA location events without valid address and 2) Non-online events with registration info
- [ ] Review and learn from content audits ^
- [ ] Schedule time with Dave and Wes to poke holes in design and/or get design sign-off
- [ ] Schedule time with Erika, Cindy, Edmund to poke holes in design and/or talk through editor work-flow and questions

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [x] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by fmccaf1** [comment]
Did request audits from CMS folks but heard from Cindy T. that this work is not being prioritized right now. Almost definitely not going to receive audits by 9/11 (end of PTEMS) 

One of the major guesses I must move forward with in the design is that non-online events do NOT have registration, while online events do sometimes have a registration.

**Comment by fmccaf1** [comment]
### To illustrate the need for required fields for location

This event is not displaying the "where" field in the events list view:
![image](https://user-images.githubusercontent.com/106624475/188901259-70086735-c78a-4445-8891-0858f8a5c515.png)

That's because the editor that created the event put address and location info in the rich text field, instead of using the location fields to make a selection and add details:
![image](https://user-images.githubusercontent.com/106624475/188901365-ca44aa17-f447-4ace-82b6-be5f7a38e150.png)

**Comment by fmccaf1** [comment]
Created [github file](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/teams/vsa/teams/public-websites/outreach-and-events/event-location-editor-ux) to describe work done so far and work to be completed. 

[First pass at designs are located here in Figma.](https://www.figma.com/file/Rj1khlzHE2VmH4aRVeAzR2/Events-near-me%3A-Editor-UX-for-testing?node-id=0%3A1)

**Comment by jilladams** [comment]
Thank you! Since a new person will need to reboot this for the remaining ACs, I'm unassigning / moving back to "New/Unrefined".

**Comment by davidmpickett** [comment]
@jilladams I didn't see this on the Q4 priorities. Should this be iceboxed?

**Comment by jilladams** [comment]
Probably - @wesrowe thoughts re: iceboxing this til it gets prioritized?

**Comment by Agile6MSkinner** [comment]
@davidmpickett Any use in keeping this ticket around?

**Comment by FranECross** [comment]
@Agile6MSkinner Regarding your comment above to Dave, just a quick note to let you know you don't need to groom Public Websites tickets. I'm working with Michelle and going through all Events tickets in preparation for Events v3. I'm on top of PW tickets and initiatives I'm assigned to, and having to PMs in the mix will confuse things. Thanks! cc @jilladams

**Comment by Agile6MSkinner** [comment]
@FranECross Yeah, my bad. I was just slamming through tickets several weeks ago and forgot to sort by facilities
